chef-provisioning-aws.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/opscode/chef-provisioning-aws'
 
   s.add_dependency 'chef', '>= 11.16.4'
-  s.add_dependency 'chef-provisioning', '~> 0.9'
+  s.add_dependency 'chef-provisioning'
```

Comment:
You should be able to rebase this off master to get the latest dependency pinning